### ISSUE TITLE
PastMassBalance is a bad name

### ISSUE THREAD

**Issue by fmaussion** [OGGM/oggm#1097]
We also use this model for GCM data...

**Comment by anoukvlug** [comment]
I agree. Is this issue intended for brainstorming for a better name for the PastMassBalance model? So far the only suggestion I thought of, is renaming it to `TransientMassBalance` to avoid confusion, but also keeping it in use as `PastMassBalance` (at least for some time) for the current users.

I wouldn't mind making a PR, if there is being agreed on making a change to the name.

**Comment by fmaussion** [comment]
yes. Other names:

- ClimaticMassBalance
- TemperatureIndexMassBalance
- TIMassBalance

I like to name the model in the name, because when several models will be available this will be necessary anyway...

**Comment by matthiasdusch** [comment]
I like TransientMassBalance, sounds so elegant. But good point about future proofness. Same is kind-off also true for ClimaticMB, this might become to general at some point as well. 
TemperatureIndexMassBalance is clear and precise. Sure it is long but I would favour this over the cryptic TIMassBalance.

**Comment by anoukvlug** [comment]
I agree, for the future it is indeed better to name the model type. I like both TemperatureIndexMassBalance and TIMassBalance, with a slight preference for the shorter TIMassBalance.  The abbreviation for Temperature Index Model, is I think easy enough to figure out and remember (, but maybe I am quite biased...).

**Comment by fmaussion** [comment]
`TIMassBalance`, short: Tim, is nice!

**Comment by anoukvlug** [comment]
I forgot a little bit about this issue.... @fmaussion, shall I make a PR to change the name to `TIMassBalance` before the 1.4 release?

**Comment by fmaussion** [comment]
Thanks @anoukvlug ! But I think it's too big of a change for now, as all the tutorials, etc. also need an update, and we need a deprecation cycle, etc. See for example: https://github.com/OGGM/oggm/blob/master/oggm/core/gcm_climate.py
